pkg/bloomgateway/processor.go

```diff
@@ -145,6 +145,11 @@ func (p *processor) processBlock(_ context.Context, bq *bloomshipper.CloseableBl
 		return err
 	}
 
+	// We require V3 schema
+	if !schema.IsCurrentSchema() {
```

Comment:
Seems like this might go stale somewhat quickly. Should we expose the schema version and have this check for `schema.Version() < V3` like you have in `SeriesWithMeta.Decode`?